sql/parser/builtins.go

```diff
@@ -943,6 +961,89 @@ func datumToString(datum Datum) (string, error) {
 	return "", fmt.Errorf("argument type unsupported: %s", datum.Type())
 }
 
+func regexpReplace(s, pattern, to, flags string) (Datum, error) {
+	pattern, global := regexpEvalFlags(pattern, flags)
+
+	patternRe, err := regexp.Compile(pattern)
+	if err != nil {
+		return DNull, err
```

Comment:
this should probably replace `nil, err` since we expect the value never to be used.